run/GEOS/GEOSCHEMchem_GridComp.rc

```diff
@@ -136,12 +136,22 @@ Use_ozonopause: -999.0
 #
 Prescribe_strat_H2O: 1
 
+# %%% Settings for excluding species from being transported
```

Comment:
Are any of the new settings options we might want for GCHP?

Reply:
This is still pretty experimental and I wouldn't include it in GCHP for now.